mautic/mautic
# Use empty value instead of constraints for required saml attributes

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/3200
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
The configuration cannot be saved unless bogus data is populated for the new SAML  attribute fields. This PR fixes that

#### Steps to test this PR:
1. Edit the configuration and go to User/Authentication Settings. Erase the Email, first name and last name values. Save. No error should result. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Save without filling in a value for the saml attributes and the form will fail validation.
